requirements.txt

```diff
@@ -2,3 +2,6 @@
 # For Django versioning we use ~= (e.g. Django~=3.2.0) which will keep the minor version up to date (e.g. the latest version prior to 3.3.0)
 
 Django~=3.2.0
+pytz~=2021.1
+Pillow~=8.3.0
+djangorestframework~=3.12
```

Comment:
I think this won't work (necessarily) - rest framework doesn't have a version compatible with 3.2 yet, same prob. I've got with the DASP site